### PR TITLE
Issue with k8s.io/docs/concepts/workloads/controllers/statefulset/

### LINKED ISSUE #15979: Improve wording for StatefulSet limitations around storage
**This is a Bug Report**

**Problem:**
Third bullet in the "limitations" section says "Deleting and/or scaling a StatefulSet down will not delete the volumes associated with the StatefulSet".
It is not wrong, but not totally right. I think it should better refer to the PVCs created from the statefulset's volumeClaimTemplate than to the pre-provisioned or dynamically created PVs boudn to those PVCs (note also that removing/not removing those PVs is really related to the PV/StorageClass set reclaimPolicy .. once PVC and PV are unbound, of course)

**Proposed Solution:**
Following text proposed for that third bullet in the "statefulset - limitations" subsection (changed text remarked with bold letters)

- Deleting and/or scaling a StatefulSet down will not delete the **persitent volumes claim objects created from any volumeClaimTemplates included in the StatefulSet**. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources. 

**Page to Update:**
https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations

### PR DESCRIPTION
**Pull request by medmedchiheb**
change suggested text for volumes

Fixes #15979

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ provides a set of stateless replicas. Controllers such as
 
 * StatefulSet was a beta resource prior to 1.9 and not available in any Kubernetes release prior to 1.5.
 * The storage for a given Pod must either be provisioned by a [PersistentVolume Provisioner](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/persistent-volume-provisioning/README.md) based on the requested `storage class`, or pre-provisioned by an admin.
-* Deleting and/or scaling a StatefulSet down will *not* delete the volumes associated with the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
+* Deleting and/or scaling a StatefulSet down will *not* delete the persitent volumes claim objects created from any volumeClaimTemplates included in the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
```
Comment (medmedchiheb) [inline content/en/docs/concepts/workloads/controllers/statefulset.md]: typo `persitent `

File: content/en/docs/concepts/workloads/controllers/statefulset.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ provides a set of stateless replicas. Controllers such as
 
 * StatefulSet was a beta resource prior to 1.9 and not available in any Kubernetes release prior to 1.5.
 * The storage for a given Pod must either be provisioned by a [PersistentVolume Provisioner](https://github.com/kubernetes/examples/tree/{{< param "githubbranch" >}}/staging/persistent-volume-provisioning/README.md) based on the requested `storage class`, or pre-provisioned by an admin.
-* Deleting and/or scaling a StatefulSet down will *not* delete the volumes associated with the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
+* Deleting and/or scaling a StatefulSet down will *not* delete the persistent volumes claim objects created from any volumeClaimTemplates included in the StatefulSet. This is done to ensure data safety, which is generally more valuable than an automatic purge of all related StatefulSet resources.
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/statefulset.md]: How about:
```suggestion
* If you delete a StatefulSet or make it have fewer replicas, your cluster will *not* delete any PersistentVolumeClaim objects that belonged to those removed replicas. This approach helps avoid deleting data that you still want to keep.
```
or some words along those lines?

Also, @medmedchiheb, you could if you wish mention a workaround / enhancement for anyone who does want to automate the deletion of those PVCs: use an appropriate Operator.

**Comment by zparnold** [comment]
/close
@medmedchiheb when you're ready to continue with this, please re-open. We'd love to have the contribution!

**Pull request closed without merging**